ui/src/cohort.ts

```diff
@@ -337,7 +337,7 @@ export function createCriteria(
 export function getCriteriaPlugin(
   criteria: tanagraUI.UICriteria,
   entity?: string
-): CriteriaPlugin<object> {
+): CriteriaPlugin<string> {
```

Comment:
Is this because you're now passing around the serialized string instead of the config or data objects taht used to be defined in the plugin ts class?

Reply:
Yes. I'm considering the possibility of switching back to passing around the internal data object and only converting to/from protos right at the edge of API class. That would require some more structural changes though and I wanted to keep that separate.